src/content/concepts/entry-points.md

```diff
@@ -126,3 +126,74 @@ __What does this do?__ We are telling webpack that we would like 3 separate depe
 __Why?__ In a multi-page application, the server is going to fetch a new HTML document for you. The page reloads this new document and assets are redownloaded. However, this gives us the unique opportunity to do things like using [`optimization.splitChunks`](/configuration/optimization/#optimizationsplitchunks) to create bundles of shared application code between each page. Multi-page applications that reuse a lot of code/modules between entry points can greatly benefit from these techniques, as the number of entry points increases.
 
 T> As a rule of thumb: Use exactly one entry point for each HTML document.
+
+### EntryDescription object
+
+An object with entry point description. You can specify the following properties.
+
+- `dependOn` :  The entrypoints that the current entrypoint depend on.They must be loaded when this entrypoint is loaded.
+- `filename` :  Specifies the name of each output file on disk.
+- `import`   :  Module(s) that are loaded upon startup.
+- `library`  :  Options for library.
+- `runtime`  :  The name of the runtime chunk. If set, a runtime chunk with this name is created else an existing entrypoint is used as runtime.
```

Comment:
```suggestion
- `runtime`  :  The name of the runtime chunk. If set, a runtime chunk with this name is created otherwise an existing entrypoint is used as runtime.
```